src/widget/wwaveformviewer.cpp

```diff
@@ -83,7 +85,11 @@ void WWaveformViewer::mousePressEvent(QMouseEvent* event) {
             auto cueAtClickPos = getCuePointerFromCueMark(m_pHoveredMark);
             if (cueAtClickPos) {
                 m_pCueMenuPopup->setTrackAndCue(currentTrack, cueAtClickPos);
-                m_pCueMenuPopup->popup(event->globalPos());
+                QPoint cueMenuTopLeft = mixxx::widgethelper::mapPopupToScreen(
+                        windowHandle()->screen()->size(),
+                        event->globalPos(),
+                        m_pCueMenuPopup->size());
+                m_pCueMenuPopup->popup(cueMenuTopLeft);
```

Comment:
You don't need to bother going back and changing this commit, but for the future please make merge commits the bare minimum required to resolve conflicts and get it to build and run, then add new changes in following commits.

Reply:
Got that. Shouldn't have combined it into the merge resolution commit. Will keep that in mind.